tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -611,6 +611,22 @@ pfcwd/test_pfc_config.py::TestPfcConfig::test_forward_action_cfg:
      conditions:
         - "asic_type in ['cisco-8000']"
 
+pfcwd/test_pfcwd_warm_reboot.py:
```

Comment:
Please make all these changes specific to cisco platform for the time being

Reply:
@neethajohn , Ack. I will change them to cisco-8000 only.

Reply:
@rraghav-cisco, is there an ETA on when the replacement testcases will be available? These are really important testcases and not having them covered is a major test gap

Reply:
@neethajohn , We are looking at the ixia tests for comparison. We are yet to complete the comparison.

From my preliminary check, the following tests are not covered. The rest are covered.
1. pfcwd/test_pfcwd_timer_accuracy.py,
2. pfcwd/test_pfcwd_all_port_storm.py and
3. qos_sai testcases.

Reply:
@neethajohn,
We plan to provide a PR for all-port storm : 3/31.
timer-accuracy: Has dependency on ixia, hope to pick it up on 4/7.

Also, until the new PRs are merged, can we keep the xfail ? Once the new PRs are merged, we can mark the corresponding tests as skipped.